datasets/ru/egrul/crawler.py

```diff
@@ -715,6 +716,9 @@ def crawl_archive(context: Context, blob_name: str) -> None:
 
 
 def crawl(context: Context) -> None:
+    # Shut up warnings being logged, we only care about them once they get pulled
+    # into our default dataset by ext_ru_egrul
+    zavod.logs.get_logger("zavod.runtime.cleaning").setLevel(logging.ERROR)
```

Comment:
I've been thinking, if we change the logging we might not think to update these log disables, and end up spamming ourselves (and possibly exhausting sentry quotas until we pay more or something) - missing critical feedback from crawlers.

I wonder if there's a way to make it more robust to such changes

Reply:
This feels smelly to me. What I'd prefer is that we exempt specific classes of FtM validation warnings - the easiest is by pre-cleaning values like:

```
from followthemoney.types import registry

def crawl():
  ...
  address = node.text
  address_clean = registry.address.clean_text(address)
  if address_clean is not None:
    entity.add("address", address_clean, original_value=address)
```

Hope that makes sense?

Reply:
That implies 9 of those in this crawler, I believe. Just checking you're ok with that, or if that sways us to try harder to make "expempt specific classes" cleaner, or try other solutions?

https://github.com/opensanctions/opensanctions/pull/1871

Reply:
I'm just really worried about messing with the logging system on a crawler-by-crawler basis, perhaps we can take that option off the menu. We could still think about adding a `mute_type_warnings` array to the dataset YAML, or make an argument to `.add(mute=True`?